website/docs/d/mssql_managed_database.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Database"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_mssql_managed_database"
+description: |-
+  Gets information about an existing Azure SQL Azure Managed Database.
+---
+
+# Data Source: azurerm_mssql_managed_database
+
+Use this data source to access information about an existing Azure SQL Azure Managed Database.
+
+## Example Usage
+
+```hcl
+data "azurerm_mssql_managed_database" "example" {
+  name                  = "example"
+  resource_group_name   = azurerm_resource_group.example.name
+  managed_instance_name = azurerm_mssql_managed_instance.example.name
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `managed_instance_name` - (Required) The name of the Managed Database.
```

Comment:
Can we list this under the Attributes Reference below since this is no longer an argument
```suggestion
* `managed_instance_name` - (Required) The name of the Managed Instance.
```